source/_integrations/teslemetry.markdown

```diff
@@ -11,82 +12,105 @@ ha_codeowners:
   - '@Bre77'
 ha_domain: teslemetry
 ha_platforms:
+  - button
   - climate
-  - diagnostics
   - sensor
 ha_integration_type: integration
 ---
 
-The Teslemetry integration exposes various commands and sensors from the Tesla vehicles connected to a [Teslemetry](https://teslemetry.com/) subscription.
+The Teslemetry integration exposes various commands and sensors from the Tesla vehicles and energy sites connected to a [Teslemetry](https://teslemetry.com/) subscription.
 
 ## Prerequisites
 
-You must have a [Teslemetry](https://teslemetry.com/) account and [access token](https://teslemetry.com/console).
+You must have a [Teslemetry](https://teslemetry.com) account, active subscription, and [access token](https://teslemetry.com/console).
+
+Vehicle delivered in 2024 and later will require a [virtual key](https://teslemetry.com/docs/topics/virtualkey) to be configured in order to run certain commands.
 
 {% include integrations/config_flow.md %}
 
 ## Entities
 
-### Climate
```

Comment:
Was the climate section removed on purpose?

Reply:
Yes, it's a table now, which should have Climate in it

Reply:
I see. The reason I asked was because I no longer found the different modes, such as the *Dog mode*.

Reply:
That's a really good point. I might just have to add a column for extras or break those out specifically.

FYI there are ~5 other PRs that I'll have to rebase with these changes you suggested.

Reply:
Right. Sorry for that. The reason I changed those headers is because we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headers.

Reply:
Haha, ok. Thank you for the heads up.